libnavigator/src/main/java/com/mapbox/navigation/navigator/internal/NavigatorLoader.kt

```diff
@@ -34,7 +34,7 @@ internal object NavigatorLoader {
             deviceProfile.customConfig
         )
         val runLoopExecutor = RunLoopExecutorFactory.build()
-        val historyRecorder = HistoryRecorderHandle.build(config)
+        val historyRecorder = HistoryRecorderHandle.build("", config)
```

Comment:
Just to be able to build with the latest NN's master, will be removed in final PR.